### PR TITLE
Allows nullable error attributes in GraphqlErrorBuilder

### PR DESCRIPTION
**Pull request by bbakerman**
See https://github.com/graphql-java/graphql-java/issues/2637

### CODE REVIEW

File: src/main/java/graphql/GraphqlErrorBuilder.java [hunk 1]
```diff
@@ -59,22 +59,32 @@ public GraphqlErrorBuilder message(String message, Object... formatArgs) {
     }
 
     public GraphqlErrorBuilder locations(List<SourceLocation> locations) {
-        this.locations.addAll(assertNotNull(locations));
+        if (locations != null) {
```
Comment (andimarek) [inline src/main/java/graphql/GraphqlErrorBuilder.java:62]: can we add the Nullable annotation to the argument to make this more clear?

**Review by andimarek — CHANGES_REQUESTED** [review]
I think we should use the Nullable annotation to communicate this more clearly

File: src/main/java/graphql/GraphqlErrorBuilder.java (hunk 1 above)
Comment (andimarek) [reply src/main/java/graphql/GraphqlErrorBuilder.java:89]: @bbakerman I think that is a missunderstanding: the result is never null, but the argument. So the directive should be on the argument.

Comment (bbakerman) [reply src/main/java/graphql/GraphqlErrorBuilder.java:88]: done

**Review by andimarek — APPROVED** [review]

**Pull request merged**